### DISCUSSION TITLE
Is there a two-way TLS (mTLS) support plan?

### DISCUSSION THREAD

**Discussion by hulinux** [directus/directus#16034] [Feature Requests]
I didn't find this described in the project documentation, can you consider adding this feature?

**Comment by rijkvanzanten** [comment]
I recommend running Directus behind a proxy loadbalancer which in turn handles TLS 👍🏻

**Reply by hulinux** [reply]
It's a right solution, very good!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
